src/fishbmc_addon.h

```diff
@@ -39,13 +39,13 @@ class ATTR_DLL_LOCAL CVisualizationFishBMC : public kodi::addon::CAddonBase,
   CVisualizationFishBMC();
   ~CVisualizationFishBMC() override;
 
-  bool Start(int channels, int samplesPerSec, int bitsPerSample, std::string songName) override;
+  bool Start(int channels, int samplesPerSec, int bitsPerSample, std::string songName);
   void Stop() override;
   void Render() override;
   void AudioData(const float* audioData,
```

Comment:
```suggestion
  void AudioData(const float* audioData, size_t audioDataLength) override;
```

ref: https://github.com/xbmc/xbmc/blob/9fc8af5d9cce4cc96c3ca7471049cc538f56b80f/xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/Visualization.h#L496

Note that this will require changes to `fishbmc_addon.cpp` also.